api/src/main/java/org/pmiops/workbench/db/dao/WorkspaceFreeTierUsageDao.java

```diff
@@ -14,6 +13,7 @@
 @Repository
 public interface WorkspaceFreeTierUsageDao extends CrudRepository<DbWorkspaceFreeTierUsage, Long> {
 
+  @Nullable
```

Comment:
I happened to see this.  Normally I'd prefer to switch this call to an Optional, but I felt that would be a little too intrusive for a PR that is not intended to make any meaningful changes